frontend/app/routes/_public+/apply+/$id+/review-information.tsx

```diff
@@ -254,15 +260,18 @@ export default function ReviewInformation() {
         </DescriptionListItem>
       </dl>
       <h2 className="mb-5 mt-8 text-2xl font-semibold">{t('apply:review-information.submit-app-title')}</h2>
-      <p className="mb-4">{t('apply:review-information.submit-p-one')}</p>
-      <p className="mb-4">{t('apply:review-information.submit-p-two')}</p>
-      <h3 className="font-semibold">{t('apply:review-information.print-app-title')}</h3>
-      <p className="mb-4">{t('apply:review-information.submit-p-three')}</p>
-      <Form method="post">
-        <Button id="confirm-button" variant="green">
-          {t('apply:review-information.submit-button')}
-        </Button>
-      </Form>
+      <p className="mb-4">{t('apply:review-information.submit-p-proceed')}</p>
+      <p className="mb-4">{t('apply:review-information.submit-p-false-info')}</p>
+      <div className="flex flex-wrap items-center gap-3">
+        <ButtonLink to={`/apply/${id}/exit-application`} variant="alternative">
+          <span className="font-semibold">{t('apply:review-information.exit-button')}</span> <FontAwesomeIcon icon={faX} className="h-3 w-3 pl-2" />
```

Comment:
The text font-weight of our button is set to normal (400). However, Figma does not align with the button font-weight in the GC Design System. If we need to implement font-semiebold for our buttons, we'll make the adjustment within the Button and ButtonLink components. Update icon's className to match others.

https://design-system.alpha.canada.ca/en/components/button/

```suggestion
          {t('apply:review-information.exit-button')}
          <FontAwesomeIcon icon={faX} className="ms-3 block size-4" />
```

Reply:
Icon `className` has not been changed.

Reply:
Should the `className` be changed? 

The icon here is an `X` as the button says "Exit application". 
Icons for other buttons are chevrons `<` or `>` because we are going backward or forward in the application.

Reply:
Sorry I misread the comment... the `className` should indeed be removed.